apps/wallet-dashboard/app/(protected)/staking/page.tsx

```diff
@@ -98,7 +156,9 @@ function StakingDashboardPage(): JSX.Element {
                     setSelectedValidator={setSelectedValidator}
                 />
             )}
-        </>
+        </Panel>
+    ) : (
+        <StartStaking />
```

Comment:
can you please add in `StartStaking` a gap between text and button :pray: 
`<div className="flex h-full w-full flex-col justify-between gap-y-md p-lg">` 
![image](https://github.com/user-attachments/assets/2fe11ff9-24f3-4eef-991f-46f2cd7d6f14)